### ISSUE TITLE
ndndpdk-depends.sh assumes/requires DPDK headers to be installed in `/usr/local/include`

### ISSUE THREAD

**Issue by Pesa** [usnistgov/ndn-dpdk#23]
It probably shouldn't assume a specific filesystem location. Another common install path is `/usr/include/dpdk`. Maybe, in addition to checking in `/usr/local`, use pkg-config to detect things, since that's precisely its purpose?

**Comment by Syafaatdehaf** [comment]
I had several problems, one of which was when I did ndnping, when I created create eth port, and when I did the command it had the output "graphql: 19 no such device".

The Ubuntu that I use is version ubuntu-20.04. I did the NDN-DPDK installation using the ndndpdk-depends.sh script to install the dependencies, then I did the build step 1 to 4, I managed to listen to tcp and then I also managed to do "started NDN -DPDK service on 127.0.0.1:3030"

May i have your solution of my problems, sir?
